src/picologging/logrecord.cxx

```diff
@@ -244,6 +251,11 @@ PyObject* LogRecord_dealloc(LogRecord *self)
     return nullptr;
 }
 
+int LogRecord_init(LogRecord *self, PyObject *args, PyObject *kwds)
+{
+    return 0;
```

Comment:
Ideally we could have some part of that logic in initizializer, but I couldn't decide what to keep and what to move to constructor.
I tried this ref: https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_new but most of attributes seem mutable in this case.

Reply:
I'll move the file path cache lookup code, but I'm changing it in another PR, so I'll do that later otherwise it'll cause a merge conflict now